.github/workflows/draft-new-release.yml

```diff
@@ -34,13 +34,22 @@ jobs:
         with:
           version: ${{ github.event.inputs.version }}
 
+      - name: Bump version in Cargo.toml
+        uses: thomaseizinger/set-crate-version@1.0.0
+        with:
+          version: ${{ github.event.inputs.version }}
```

Comment:
This will not commit it right?

Reply:
This modifies the version in the file, adding and committing the changes is in the lines below. 

Note: I did not specifically check the code of the action but configured it according to the xmr-btc project where we set it up like this.